### PR TITLE
ELEMENTS-750: cleanup left over loading spinner + fix display loading…

### PR DESCRIPTION
**Pull request by guirenard**
… message

### CODE REVIEW

**Comment by nmpcunha** [comment]
👍

File: nuxeo-data-table/iron-data-table.html [hunk 1]
```diff
@@ -365,7 +348,7 @@ <h2>[[i18n('command.add')]]</h2>
        */
       class DataTable
         extends Polymer.mixinBehaviors([Polymer.IronResizableBehavior, Polymer.IronFormElementBehavior,
-          Polymer.IronValidatableBehavior, Nuxeo.PageProviderDisplayBehavior, Nuxeo.I18nBehavior,
+          Polymer.IronValidatableBehavior, Nuxeo.PageProviderDisplayBehavior,
```
Comment (Gabez0r) [inline nuxeo-data-table/iron-data-table.html]: We should remove the `* @appliesMixin Nuxeo.I18nBehavior` from the documentation of this element as well. (line 343)

**Comment by Gabez0r** [comment]
👍

File: nuxeo-data-table/iron-data-table.html (hunk 1 above)
Comment (guirenard) [reply nuxeo-data-table/iron-data-table.html]: :+1:

**Pull request merged**